packages/components/src/components/tab-header/tab-header.tsx

```diff
@@ -65,9 +69,9 @@ export class TabHeader {
     return (
       <Host
         id={`scale-tab-header-${this.generatedId}`}
-        role="tab"
+        role={this.disabled ? false : 'tab'}
         aria-selected={String(this.selected)}
```

Comment:
On disabled tabs we get an `aria-selected="undefined"`. I think changing this line to `aria-selected={this.selected ? 'true' : 'false'}` will fix this. What do you think?

Reply:
Hi,
yup I agree that `aria-selected="undefined" ` is not that nice. I'll change it in a sec :)

(no worries about the delay ^^ It's vacation time)

Reply:
Ok, this breaks the unit tests. I need to take a closer look

Reply:
Probably because the snapshots already had "undefined" in them?

```diff
- <scale-tab-header aria-selected="undefined" id="scale-tab-header-0" role="tab" tabindex="-1">
+ <scale-tab-header aria-selected="false" id="scale-tab-header-0" role="tab" tabindex="-1">
```

Maybe `yarn test --spec --updateSnapshot` solves it…

Reply:
Looking forward to what you find out. Thanks for the update :)

Reply:
Jup that's it. I will update the Snapshot